Makefile

```diff
@@ -74,6 +74,9 @@ deploy-frontend-heroku:
 send-test-email-heroku:
 	make manage-heroku CMD="send_test_mail ${EMAIL}"
 
+init-behave:
+	cat features/data/fixtures.sql| docker-compose exec -T db psql -U pah-fm
```

Comment:
I am getting 
```
cat features/data/fixtures.sql| docker-compose exec -T db psql -U pah-fm
cat: features/data/fixtures.sql: No such file or directory
```
It probably should say
`cat pah_behave/pah_behave/features/data/fixtures.sql| docker-compose exec -T db psql -U pah-fm`

Reply:
fixed